core/common/src/ipc/IpcWebSocketTransport.ts

```diff
@@ -98,7 +98,7 @@ function reviver(_key: string, value: any) {
 }
 
 function replaceBinary(value: any): Marker | undefined {
-  if (ArrayBuffer.isView(value) || Buffer.isBuffer(value)) {
+  if (ArrayBuffer.isView(value) || PlatformUtilities.utilities.isBackendBuffer(value)) {
     const index = parts.push(value) - 1;
```

Comment:
this is only for backend, right? This class is internal, can't you can split it out of common into frontend and backend classes?

Reply:
No, that class is the transport functionality that is actually common to both. There are frontend and backend derivatives.

Reply:
I know, i'm saying split it into two classes since it's @internal.

Reply:
We already have three classes in this case (common transport and backend/frontend derivatives).
I can just pull the functionality on line 101 into the backend one.